client/src/components/Footer/index.js

```diff
@@ -6,8 +6,8 @@ const Footer = () => (
   <footer className="footer">
     <ul className="footer__list">
       <li className="footer__item">&copy;SitSpot 2019</li>
-      <li className="footer__item">Privacy and Cookies</li>
-      <li className="footer__item">Terms</li>
+      {/* <li className="footer__item">Privacy and Cookies</li>
+      <li className="footer__item">Terms</li> */}
```

Comment:
pls do not commit commented code, either remove it or don't

Reply:
it was removed from the prototype, but these pages are still there in the stretch goal, so i just commented

Reply:
then it's a good practice to put a TODO comment explaining that - not everybody is aware of the situation, plus your goals might change and then it's gonna be left there and nobody will know why it's commented